hw/ip/otbn/rtl/dv/model/otbn_model.cc

```diff
@@ -55,6 +56,7 @@ int run_model(const char *imem_scope, int imem_words, const char *dmem_scope,
   std::memcpy(ifname, dir, strlen(dir));
   std::memcpy(dfname, dir, strlen(dir));
   std::memcpy(cfname, dir, strlen(dir));
+  std::memcpy(tfname, dir, strlen(dir));
```

Comment:
Wow, I hadn't seen this trick before. Kind of icky, but very neat!

Would it make more sense to change the model to take an output directory, rather than a list of positional arguments for all its output files? One plus would be that the C driver and the Python model would be much more loosely coupled.

Reply:
Yeah, I was thinking the same. Will change it in the next sprint, thanks!